compiler/record-minmax/src/MinMaxObserver.cpp

```diff
@@ -86,6 +86,10 @@ void MinMaxObserver::postTensorWrite(const luci::CircleNode *node,
     if (isnan(number))
       continue;
 
+    // TODO use metadata hints to detect such cases
```

Comment:
@jinevening 
This is what I was talking about in https://github.com/Samsung/ONE/pull/7236/files#r687530696

What do you think about adding hints in circle metadata to control such hacks?

Reply:
It looks ok. The lowest value of float may not appear in normal execution.